src/apps/backend/app.ts

```diff
@@ -1,22 +1,21 @@
+import 'module-alias/register';
 import { Server } from 'http';
 import * as path from 'path';
 
-import cors from 'cors';
-import express, { Application } from 'express';
-import expressWinston from 'express-winston';
-
-import { AccountServer } from './modules/account';
-import { ApplicationServer } from './modules/application';
-import { AuthenticationServer } from './modules/authentication';
-import { ConfigService } from './modules/config';
+import { AccountServer } from 'backend/modules/account';
+import { ApplicationServer } from 'backend/modules/application';
+import { AuthenticationServer } from 'backend/modules/authentication';
+import { ConfigService } from 'backend/modules/config';
 import {
   DocumentationServer,
   DocumentationService,
-} from './modules/documentation';
-import { expressListRoutes } from './modules/documentation/list-routes';
-import { Logger, CustomLoggerTransport } from './modules/logger';
-import { PasswordResetTokenServer } from './modules/password-reset-token';
-import { TaskServer } from './modules/task';
+} from 'backend/modules/documentation';
+import { expressListRoutes } from 'backend/modules/documentation/list-routes';
```

Comment:
We should import `expressListRoutes` in index.ts of `backend/modules/documentation`. This way there is no need to look beyond `backend/modules/documentation`